lib/web_ui/dev/test_runner.dart

```diff
@@ -98,54 +100,75 @@ class TestCommand extends Command<bool> {
       'test',
     ));
 
-    // Separate screenshot tests from unit-tests. Screenshot tests must run
-    // one at a time. Otherwise, they will end up screenshotting each other.
-    // This is not an issue for unit-tests.
-    final FilePath failureSmokeTestPath = FilePath.fromWebUi(
-      'test/golden_tests/golden_failure_smoke_test.dart',
-    );
-    final List<FilePath> screenshotTestFiles = <FilePath>[];
-    final List<FilePath> unitTestFiles = <FilePath>[];
-
-    for (io.File testFile
-        in testDir.listSync(recursive: true).whereType<io.File>()) {
-      final FilePath testFilePath = FilePath.fromCwd(testFile.path);
-      if (!testFilePath.absolute.endsWith('_test.dart')) {
-        // Not a test file at all. Skip.
-        continue;
-      }
-      if (testFilePath == failureSmokeTestPath) {
-        // A smoke test that fails on purpose. Skip.
-        continue;
+    // Screenshot tests and smoke tests only run in Chrome.
+    if (isChrome) {
+      // Separate screenshot tests from unit-tests. Screenshot tests must run
+      // one at a time. Otherwise, they will end up screenshotting each other.
+      // This is not an issue for unit-tests.
+      final FilePath failureSmokeTestPath = FilePath.fromWebUi(
+        'test/golden_tests/golden_failure_smoke_test.dart',
+      );
+      final List<FilePath> screenshotTestFiles = <FilePath>[];
+      final List<FilePath> unitTestFiles = <FilePath>[];
+
+      for (io.File testFile
+          in testDir.listSync(recursive: true).whereType<io.File>()) {
+        final FilePath testFilePath = FilePath.fromCwd(testFile.path);
+        if (!testFilePath.absolute.endsWith('_test.dart')) {
+          // Not a test file at all. Skip.
+          continue;
+        }
+        if (testFilePath == failureSmokeTestPath) {
+          // A smoke test that fails on purpose. Skip.
+          continue;
+        }
+
+        if (isChrome &&
```

Comment:
According to the `if` statement above `isChrome` is always `true` here.

Reply:
Thanks! Done. 

the code above used to be different.